docs/content/app-protect/configuration.md

```diff
@@ -9,13 +9,13 @@ docs: "DOCS-578"
 ---
 
 This document describes how to configure the NGINX App Protect module
-> Check out the complete [NGINX Ingress Controller with App Protect example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/appprotect).
+> Check out the complete NGINX Ingress Controller with App Protect example resources on GitHub [for VirtualServer resources](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/custom-resources/waf) and [for Ingress resources](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/ingress-resources/appprotect).
 
 ## Global Configuration
 
 The NGINX Ingress Controller has a set of global configuration parameters that align with those available in the NGINX App Protect module. See [ConfigMap keys](/nginx-ingress-controller/configuration/global-configuration/configmap-resource/#modules) for the complete list. The App Protect parameters use the `app-protect*` prefix.
 
-> Check out the complete [NGINX Ingress Controller with App Protect example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/appprotect).
+> Check out the complete NGINX Ingress Controller with App Protect example resources on GitHub [for VirtualServer resources](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/custom-resources/waf) and [for Ingress resources](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/ingress-resources/appprotect).
```

Comment:
I'm adding the comment here, but it's general, should we rename the folders to `appprotect-dos` and `approtect-waf` since we're at it? 

Also, move from calling it just App Protect to App Protect WAF? 

cc @nginxinc/nginx-docs

Reply:
@lucacome agreed on both counts

Reply:
yes, absolutely.

Reply:
Will do 👍🏼

Reply:
One thing to take into consideration here is that the file path == the path to the doc in the docs website, so if you change the file path you break links and bookmarks. 

You can add the previous path to the frontmatter if you do move docs around, so Hugo will automatically redirect from the old path to the new path. 
https://gohugo.io/content-management/urls/#aliases